privacyidea/static/components/login/controllers/loginControllers.js

```diff
@@ -259,13 +259,14 @@ angular.module("privacyideaApp")
                         $scope.challenge_message = $scope.challenge_message + ' ' + multi_challenge[i].serial;
                     }
                     let challenge = multi_challenge[i];
-                    let attributes = challenge.attributes ? challenge.attributes : null;
-                    if (challenge === null || (attributes && attributes.hideResponseInput !== true)) {
-                        $scope.hideResponseInput = false;
-                    }
                     if (challenge !== null) {
+                        if (challenge.client_mode === 'interactive') {
+                            // if we have at least one interactive token, we need to shoe the input field
```

Comment:
shoe -> show

Reply:
Done in [25ed8ff](https://github.com/privacyidea/privacyidea/pull/3082/commits/25ed8ffa2fbb189a31b4c536a4610434327413de)